src/app/components/transcriptions/view-transcript/view-transcript.component.html

```diff
@@ -0,0 +1,68 @@
+<ng-container *ngIf="data$ | async as vm">
+  <app-breadcrumb>
+    <ng-container *breadcrumb="['/search']">Search</ng-container>
+    <ng-container *breadcrumb="['/case', vm.transcript.case_number]">{{ vm.transcript.case_number }}</ng-container>
```

Comment:
This needs to link to the case_id not the case number

Reply:
case_id refers to the internal case_id, with this requested change. This is how the page looks:
<img width="1098" alt="image" src="https://github.com/hmcts/darts-portal/assets/134072550/e1ddae45-f7d3-410c-a6e9-715473e5e988">
As opposed to the current version with case_number:
<img width="1340" alt="image" src="https://github.com/hmcts/darts-portal/assets/134072550/0329f4fb-8664-4a23-9fd1-657411c70cc8">

Reply:
Display the case number but link to the case id

Reply:
It does the same thing in the hearing page

Reply:
ah ok, that makes sense